### PR TITLE
Update Cypress test. Group features.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Add reset/cancel grouping. Add ungroup vis ```Shift+G```.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks_objects/case_15_group_features.js [hunk 1]
```diff
@@ -91,6 +99,21 @@ context('Group features', () => {
         cy.changeColorViaBadge(color);
     }
 
+    function checkShapesFill(equal) {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_15_group_features.js:102]: testShapesFillEquality

File: tests/cypress/integration/actions_tasks_objects/case_15_group_features.js [hunk 2]
```diff
@@ -70,12 +70,20 @@ context('Group features', () => {
...
+    function groupObjects(objectsArray, cancelGrouping) {
         cy.get('.cvat-group-control').click();
         for (const shapeToGroup of objectsArray) {
-            cy.get(shapeToGroup).click();
+            cy.get(shapeToGroup).click().should('have.class', 'cvat_canvas_shape_grouping');
         }
+        cancelGrouping ? cy.get('body').type('{Esc}') : cy.get('.cvat-group-control').click();
+    }
+
+    function unGroupObjects(objectsArray) {
```
[the remark this answers is not in the document]
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_15_group_features.js]: I agree. Removed the argument.

File: tests/cypress/integration/actions_tasks_objects/case_15_group_features.js [hunk 3]
```diff
@@ -70,12 +70,20 @@ context('Group features', () => {
         cy.saveLocalStorage();
     });
 
-    function groupObjects(objectsArray) {
+    function groupObjects(objectsArray, cancelGrouping) {
```
[the remark this answers is not in the document]
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_15_group_features.js]: This function is also used for grouping ```trackArray```

File: tests/cypress/integration/actions_tasks_objects/case_15_group_features.js [hunk 4]
```diff
@@ -123,16 +146,10 @@ context('Group features', () => {
...
-                    .then(($shapesGroupColor) => {
-                        // expected rgb(250, 50, 83) to not equal rgb(224, 224, 224)
-                        expect($shapesGroupColor).to.not.equal(defaultGroupColorRgb);
-                        shapesGroupColor = $shapesGroupColor;
-                    });
-            }
+            groupObjects(shapeArray, true); // Reset grouping
+            checkShapesFill(true);
+            groupObjects(shapeArray); // Group
+            checkShapesFill(false);
```
[the remark this answers is not in the document]
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_15_group_features.js]: Done

File: tests/cypress/integration/actions_tasks_objects/case_15_group_features.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_15_group_features.js]: Renamed

**Review by bsekachev — APPROVED** [review]

**Pull request merged**